pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -274,7 +274,7 @@ func TestPriorityQueue_AddUnschedulableIfNotPresent(t *testing.T) {
 // current scheduling cycle will be put back to activeQueue if we were trying
 // to schedule them when we received move request.
 func TestPriorityQueue_AddUnschedulableIfNotPresent_Backoff(t *testing.T) {
-	q := NewPriorityQueue(nil, nil)
+	q := NewPriorityQueueWithClock(nil, clock.NewFakeClock(time.Now()), nil)
 	totalNum := 10
 	expectedPods := make([]v1.Pod, 0, totalNum)
 	for i := 0; i < totalNum; i++ {
```

Comment:
We are silencing errors down here. Can we add t.Fatal or at least t.Error for the return value of `Add` and `AddUnschedulableIfNotPresent`

Reply:
`Add` seems to be already dump errors via klog, I will log the error for `AddUnschedulableIfNotPresent`.